portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase

```diff
@@ -638,4 +638,86 @@ definition {
 		}
 	}
 
+	@description = "Bug: LRAC-8239 | Automation ID: LRAC-11536 | Test Summary: Delete a segment in edit mode"
+	@priority = "2"
+	test DeleteSegmentInEditMode {
+		var staticSegmentName = "New Static Segment";
+		var dynamicSegmentName = "New Dynamic Segment";
+
+		task ("Create a blog and add to a page and then connect AC and DXP") {
+			ACUtils.createBlogsAndAddToPage();
+
+			ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+		}
+
+		task ("See the blog and launch AC") {
+			ACUtils.navigateToSitePage(
+				actionType = "View Blog",
+				documentTitleList = "Blogs AC Title",
+				pageName = "AC Page",
+				siteName = "Site Name");
+
+			ACUtils.closeAllSessionsAndWait();
+
+			ACUtils.launchAC();
+
+			ACProperties.switchProperty(propertyName = "${assignedPropertyName}");
+		}
+
+		task ("Go to segments and create a static segment") {
+			ACNavigation.goToSegments();
+
+			ACSegments.createStaticSegment();
+
+			ACUtils.setItemName(itemName = "${staticSegmentName}");
+
+			ACSegments.addStaticMember(tableMember = "Test Test");
+
+			ACSegments.saveSegment();
+		}
+
+		task ("Edit the static segment, don't save the changes and then delete it, assert that the segment is not in the list") {
+			ACNavigation.goToSegments();
+
+			ACSegments.editSegment(segmentName = "${staticSegmentName}");
+
+			ACSegments.removeStaticMember(tableMember = "Test Test");
+
+			ACSegments.deleteSegmentInEditMode();
```

Comment:
Instead of a new macro does it make sense to use. `ACUtils.clickAnyButton(button= "Delete Segment")` Let me know what you think.

Reply:
Or I guess maybe keep the macro but no need to have a path to use the Delete Segment button: DELETE_SEGMENT_BUTTON_EDIT_MODE

Reply:
I agree with Josh, I think maybe creating a macro that will only be used in this test may not be necessary. What do you think, @manuferreira?